content/blog/announcing-new-transform-callbacks/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Announcing new transform callbacks"
+date: 2024-05-15
+meta_desc: Transform callbacks are a rebuild of the transformations system to work for Multi-Language Components
+meta_image: meta.png
```

Comment:
FYI - this is likely failing because the meta.png `meta_image` does not yet exist.